### PR TITLE
B 20673 int

### PR DESCRIPTION
**Pull request by deandreJones**
## Summary

Will need new payment request status for TPPS RECEIVED
We send TPPS an invoice, EDI 858 with payment request information
TPPS will receive this information and send back an EDI 997
The receipt of an EDI 997 confirms that the TPPS has received the invoice
(TBD) An 824 will provide more detail in addition to the 997 for what errors, if any, came with the sending 
Based on data we receive about how TPPS pays HomeSafe (receiving the 997 (possibly receiving the 824), we will need to build backend logic for updating the status of the payment request to reflect those updates
UI portion: add the TPPS received status to PaymentRequestCard.jsx

## Verification Steps for the Author

These are to be checked by the author.

- [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
- [ ] Have the Agility acceptance criteria been met for this change?

## Verification Steps for Reviewers

These are to be checked by a reviewer.

- [ ] Has the branch been pulled in and checked out?
- [ ] Have the BL acceptance criteria been met for this change?
- [ ] Was the CircleCI build successful?
- [ ] Has the code been reviewed from a standards and best practices point of view?

### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Access the office app
2. Login as a TIO
3. approve a payment request
4. SINCE files aren't sent to get locally or integration- the next step is to find that approved payment request and update the status to 'TPPS Received' 
5. verify the status updated on the UI

### Frontend

- [ ] There are no aXe warnings for UI.
- [ ] This works in [Supported Browsers and their phone views](https://transcom.github.io/mymove-docs/docs/adrs/Browser-Support/#minimum-browser-requirements) (Chrome, Firefox, Edge).
- [ ] There are no new console errors in the browser devtools.
- [ ] There are no new console errors in the test output.
- [ ] If this PR adds a new component to Storybook, it ensures the component is fully responsive, OR if it is intentionally not, a wrapping div using the `officeApp` class or custom `min-width` styling is used to hide any states the would not be visible to the user.
- [ ] This change meets the standards for [Section 508 compliance](https://www.ssa.gov/accessibility/andi/help/install.html).

### Backend

- [ ] Code follows the guidelines for [Logging](https://transcom.github.io/mymove-docs/docs/getting-started/development/logging).
- [ ] The requirements listed in [Querying the Database Safely](https://transcom.github.io/mymove-docs/docs/backend/guides/golang-guide#querying-the-database-safely) have been satisfied.

#### Any new migrations/schema changes:

- [ ] Follows our guidelines for [Zero-Downtime Deploys](https://transcom.github.io/mymove-docs/docs/backend/setup/database-migrations#zero-downtime-migrations).
- [ ] Have been communicated to #g-database.
- [ ] Secure migrations have been tested following the instructions in our [docs](https://transcom.github.io/mymove-docs/docs/backend/setup/database-migrations#secure-migrations).

## Screenshots
Screenshot 2024-07-29 at 11 45 59 AM

### CODE REVIEW

**Review by traskowskycaci — CHANGES_REQUESTED** [review]
Looks good!

What do you think of modifying the "Accepted on" to be "Received on" with the received_by_gex_at date, assuming that column gets populated when we get the 997? Or if not, even just changing "Accepted on" to "Received"

If you pull in this [refactor commit](https://github.com/transcom/mymove/commit/a6b6f55ec8118870d7c123563381ea7cd9c3fdfe
) and add a case for if (paymentRequestStatus === PAYMENT_REQUEST_STATUS.TPPS_RECEIVED) you should be able to customize what that text says underneath

image

**Comment by deandreJones** [comment]
> Looks good!
> 
> What do you think of modifying the "Accepted on" to be "Received on" with the received_by_gex_at date, assuming that column gets populated when we get the 997? Or if not, even just changing "Accepted on" to "Received"
> 
> If you pull in this [refactor commit](https://github.com/transcom/mymove/commit/a6b6f55ec8118870d7c123563381ea7cd9c3fdfe) and add a case for if (paymentRequestStatus === PAYMENT_REQUEST_STATUS.TPPS_RECEIVED) you should be able to customize what that text says underneath
> 
> image

updated 
image

**Review by traskowskycaci — APPROVED** [review]
Looks great!

File: swagger-def/definitions/AdditionalDocuments.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ properties:
     readOnly: true
   documentId:
     description: The id of the Document that contains all file uploads for this additional document
-    type: string
+    type: stringmake s
```
Comment (TevinAdams) [inline swagger-def/definitions/AdditionalDocuments.yaml:18]: Is this a typo?

Comment (TevinAdams) [reply swagger-def/definitions/AdditionalDocuments.yaml:18]: Is that supposed to be just string data type.

Comment (deandreJones) [reply swagger-def/definitions/AdditionalDocuments.yaml]: fixed

**Review by TevinAdams — APPROVED** [review]
Approved. Test case works and code reviewed. Went to DB and changed status of approved payment request to TPPS_RECEIVED and noticed the UI reflect that status for the payment request.

**Pull request merged**